org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -476,3 +476,47 @@ GithubOidcSageBionetworksSynapseR:
   DefaultOrganizationBinding:
     Account: !Ref SynapseProdAccount
     Region: us-east-1
+
+GithubOidcWorkflowsDevNextflowInfra:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: github-oidc-provider-access.njk
+  StackName: !Sub ${resourcePrefix}-${appName}-nextflow-infra
+  Parameters:
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ProviderRoleName: !Sub ${resourcePrefix}-${appName}-nextflow-infra
+    MaxSessionDuration: "21600"
+    ManagedPolicyArns:
+      - "arn:aws:iam::aws:policy/AdministratorAccess"
+      - "arn:aws:iam::aws:policy/AWSKeyManagementServicePowerUser"
+  TemplatingContext:
+    GitHubOrg: "Sage-Bionetworks-Workflows"
+    Repositories:
+      - name: "nextflow-infra"
+        branches: ["dev"]
+  DefaultOrganizationBinding:
+    Account:
+      - !Ref WorkflowsNextflowDevAccount
+    Region: us-east-1
+
+GithubOidcWorkflowsProdNextflowInfra:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: github-oidc-provider-access.njk
+  StackName: !Sub ${resourcePrefix}-${appName}-nextflow-infra
```

Comment:
I think these need unique names, but I'm not sure what happens if we try to re-use a stack name

Reply:
it should not be a problem because they are getting deployed to different accounts.